database/factories/TourFactory.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use App\Models\Tour;
+
+class TourFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Tour::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'destination' => $this->faker->city(),
+            'start' => $this->faker->dateTimeBetween('now', 'next month'),
+            'end' => $this->faker->dateTimeBetween('next month + 1 month', 'next month + 2 months'),
+            'price' => $this->faker->randomFloat(2),
```

Comment:
This is throwing an error on my system because I think randomFloat(2) sometimes generates a number too big for the specified datatype in the migration. Is it just an issue with my system or is it happening on yours too?

Reply:
Strange, by default DB is hardcoded to float(8,  2) as well as migration so should not be able to generate anything bigger than that. I am not getting an error.